app/styles/modules/_custom-rows.scss

```diff
@@ -86,6 +86,16 @@ ul.links {
 }
 
 .choose-what-to-sync-row {
+  &:active{
+    .fxa-checkbox__box-outline{
+      border: 2px solid $checkbox-border-color;
+
+      .fxa-checkbox__tick-outline{
+        background: url("/images/tick.svg");
```

Comment:
I'm good with that, I will see if I can also fix the label alignment in fxa-checkbox as well.